android/app/build.gradle

```diff
@@ -266,15 +274,10 @@ dependencies {
 
     compile 'com.github.status-im:function:0.0.1'
     implementation fileTree(dir: "node_modules/realm/android/libs", include: ["*.jar"])
-    compile 'com.facebook.fresco:animated-gif:1.9.0'
 }
 
 // Run this once to be able to run the application with BUCK
 // puts all compile dependencies into folder libs for BUCK to use
```

Comment:
Remove?

Reply:
nope, it seems it was moved few lines above, will move back

Reply:
fixed